ansible/roles/tower_demo/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+- name: install pip via yum
+  yum:
+   name: python-pip
+
+- name: Install Tower cli
+  pip:
+   name: 'ansible-tower-cli'
+   version: '3.1.8'
+
+- name: Configure the tower cli file
+  template:
+    src: "{{ ANSIBLE_REPO_PATH }}/files/tower_cli.j2"
+    dest: "~/.tower_cli.cfg"
+    mode: 0640
+
+- name: copy the license file
+  copy:
+   src: "{{ ANSIBLE_REPO_PATH }}/files/tower_license"
+   dest: "/tmp/tower_license"
+
+- name: copy the private key
+  copy:
+    src: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ guid + 'key' }}"
+    dest: /tmp/id_rsa
```

Comment:
in `/tmp/` ?  does it mean it won't work after a reboot ?

Reply:
this is just temporary to upload to tower